quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/GameResult.java

```diff
@@ -0,0 +1,27 @@
+package com.rl.mpquoridor.models;
+
+import com.rl.mpquoridor.models.actions.TurnAction;
+import com.rl.mpquoridor.models.players.Player;
+import com.sun.istack.NotNull;
+import org.springframework.data.util.Pair;
+
+import java.util.Collections;
+import java.util.List;
+
+public class GameResult {
+    private final Player winner;
+    private final List<Pair<Player, TurnAction>> history;
```

Comment:
I think Pair fields should be of a type consist two fields.
history should be of type PlayerTurn or HistoryAction or something like that...